### DISCUSSION TITLE
Feature idea: Improve map/timeline view

### DISCUSSION THREAD

**Discussion by rybber78** [photoprism/photoprism#2281] [Ideas]
I know you guys are busy building great stuff. But even the greatest of artists gets inspired (borrows) ideas from each other.

Take a look at the My Map Google Photos Android app feature. An amazingly handy way of viewing all the photos/timeline of photos taken within a given map area. Would be super useful way to view your images. Also has a heatmap. Currently I find the map feature useful, but when clicking an image only the actual image gets show and I can't continue browsing the other images from the same time/location easily.

https://www.groovypost.com/howto/see-your-google-photos-images-on-a-map/

**Comment by graciousgrey** [comment]
Thank you very much for your feedback :)
The places section will be improved once we have resources for this. 
There are already some related issues, such as: https://github.com/photoprism/photoprism/issues/1187.

Happy to collect more ideas in this discussion.
